src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -51,6 +58,36 @@ class SettingsLayout extends React.Component {
   }
 }
 
+const Mask = styled('div')`
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
+    display: ${p => (p.isVisible ? 'none' : 'block')};
+    background: rgba(0, 0, 0, 0.35);
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    z-index: 2;
+    animation: 0.25s ease-in-out comeIn;
+    @keyframes comeIn {
```

Comment:
There's an emotion helper for this. https://github.com/getsentry/sentry/blob/7aa676f5f62592407a26d8beeebb03384aecf0cb/src/sentry/static/sentry/app/styles/animations.tsx#L25-L41

Reply:
I was having issues passing props to the keyframes helper. @dashed found this article highlighting the bug. 
https://github.com/styled-components/styled-components/issues/397